app/journeys/probate-caveats.js

```diff
@@ -20,9 +20,10 @@ const stepList = {
     RemoveAlias: 'DeceasedOtherNames',
     DeceasedOtherNames: 'DeceasedAddress',
     DeceasedAddress: 'Summary',
-    Summary: 'EndJourneyPage',
-    AddressLookup: 'AddressLookup'
+    Summary: 'ThankYou',
+    ThankYou: 'EndJourneyPage',
```

Comment:
I think we can get rid of the endjourneyPage now, it was a simple placeholder before the Thankyou page was done.

Reply:
Deleted it